packages/documentation/pages/utilities.ts

```diff
@@ -69,6 +74,47 @@ export const createActions = (
 		  ]
 		: undefined
 
+export const createRemoteUpload = (
+	hasRemoteUpload: boolean,
+): Record<string, unknown> | undefined =>
+	hasRemoteUpload
+		? {
+				actions: {
+					/* eslint-disable no-console */
+					onCancel: (id: number | string) => console.log(`onCancel: ${id}`),
+					onDelete: (id: number | string) => console.log(`onDelete: ${id}`),
+					onRetry: (id: number | string) => console.log(`onRetry: ${id}`),
+					onUpload: (id: number | string) => console.log(`onUpload: ${id}`),
+					/* eslint-enable no-console */
+				},
+				payload: {},
+		  }
+		: undefined
+
+const createRemoteUploadCode = (component: ComponentValue): string | null => {
+	const remoteUpload = createRemoteUpload(component.hasRemoteUpload)
+
+	return remoteUpload
+		? `${[
+				...Object.entries(remoteUpload)
+					.map(([key, value]) => {
+						if (key === 'actions')
+							return [
+								`\t:${key}: {`,
+								...Object.keys(value as Record<string, unknown>).map(
+									(k) => `\t\t${k}: (id: number | string) => {},`,
```

Comment:
isn't this on Vue level 

the syntax looks like JS to me 

should be `{k}=` not `:`

Reply:
Oh no nvm - the payload itself is an object, nvm :)